helmfile/values/grafana-ops.yaml.gotmpl

```diff
@@ -75,3 +75,6 @@ dashboards:
   welcome:
     enabled: true
     user_visible: true
+  ciskubernetesbenchmark:
+    enabled: true
+    user_visible: true
```

Comment:
do we want this user visible by default at this stage?

Reply:
I have no real opinion on this.

Reply:
Yes I think it can be vidible to our users by default